server/terraform/modules/myhealth/main.tf

```diff
@@ -97,11 +98,14 @@ resource "google_firebase_project_location" "fireloc" {
 }
 
 
-# App Engine
+# App Engine - required to create the Firestore database.
+#   See: https://firebase.google.com/docs/firestore/solutions/automate-database-create#create_a_database_with_terraform
+#   NOTE: contrary to this documentation, we seem to still need a manual step to actually create the Firestore database.
+#         That manual step is documented in `server/terraform/README.md`.
```

Comment:
can it be done in gcloud?

Reply:
Surprisingly, no! Using `gcloud` calls the same APIs that Terraform does, which makes it even stranger that neither of those work! The only way I've found to do this is actually click around on the webpage.